src/runtime/web/index.js

```diff
@@ -1,6 +1,35 @@
-import { createSimpleContext } from './api';
-import { View } from '../../bindings/html/view';
+import { prioritizeLocales } from '../../intl/index';
+import { HTMLLocalization } from '../../bindings/html';
+import { documentReady, getResourceLinks, getMeta } from './util';
```

Comment:
At the same time, the `Localization` class doesn't even know about `meta` or `links` or `messagebundles`.  All it cares for is an initialization promise which resolves to a list of lazy bundles, which is the _de facto_ current language negotiation state. This means that we can use the same `HTMLLocalization` in different runtimes, e.g. web (which requires `meta`) and gecko (which doesn't because the chrome registry returns the right resource file).